Makefile

```diff
@@ -4,7 +4,8 @@
 
 TARGET = crouton
 EXTTARGET = crouton.zip
-SRCTARGETS = $(patsubst src/%.c,crouton%,$(wildcard src/*.c))
+SRCTARGETS = $(patsubst src/%.c,crouton%,$(shell find src/ -name *.c -and -not -name freon.c))
+LIBS = croutonfreon.so #croutonxorg.so
```

Comment:
Oops, xorg.c got removed a while ago, but I forgot to remove it from the Makefile.  You can eliminate the last trace of it.